src/mlpack/methods/decision_tree/decision_tree.hpp

```diff
@@ -496,6 +502,10 @@ class DecisionTree :
   std::vector<DecisionTree*> children;
   //! The dimension this node splits on.
   size_t splitDimension;
+  //! Stores the number of times a particular feature was used to split
+  map<size_t, size_t> featureFrequency;
+  //! Stores the net gain provided by a particular feature for splitting
+  map<size_t, double> featureCover;
```

Comment:
I don't have the time to review this PR deeply but I saw it touched this file, so I took a look: we can't add members to `DecisionTree`---these `map`s will be duplicated across every single node in the tree and will make the serialized model orders of magnitude larger (and predictions will be slower too).  An alternate approach will be needed.

Reply:
Oh right yes. I'll change it right away.
Thanks!